spec/integration/_dev/benchmark/spec.yml

```diff
@@ -33,6 +33,27 @@ spec:
           type: folder
           name: deploy
           $ref: "./deploy/spec.yml"
+        - description: A folder containing resources used by one or more benchmarks.
+          type: folder
+          pattern: '^[a-z0-9]+-benchmark$'
+          additionalContents: false
+          required: false
+          contents:
+            - description: A file with the config definition for the corpus generator.
+              type: file
+              pattern: '^config.yml$'
+              contentMediaType: "application/x-yaml; require-document-dashes=true"
```

Comment:
If we require document dashes we need to include them in the files. This is making CI to fail.